### DISCUSSION TITLE
vercel部署成功，但是后台无法登陆，客户端无法评论。。？

### DISCUSSION THREAD

**Discussion by runawayShrimp** [walinejs/waline#1141] [Q&A]
我用gh pages搭博客，数据库是gh private repo，部署在vercel上。地址是这个：
https://waline-azure.vercel.app
vercel显示部署是成功了。
我尝试从这里
https://waline-azure.vercel.app/ui/register
注册管理员账号，注册是成功了，数据库也能看到新用户写入，但是尝试登陆的时候却报错“账号密码错误”，无法登陆。尝试找回密码，则报错“Failed to send”。如果不登陆尝试提交评论，则报错“(unknown path) SyntaxError: Invalid or unexpected token”，但是数据库里同样能看到新评论写入。不知道是哪一步弄错了><

**Comment by lizheming** [comment]
看起来是数据解析有点问题，晚点我再 check 一下。另外不建议用 GitHub 存储方式，建议用其它的存储方式比较好。

**Comment by runawayShrimp** [comment]
谢谢回复！好的，我在planetscale上建了新的数据库，setup成功了。我想在vercel上deploy一个新的project与这个数据库关联，但是在这一步失败了：

<img width="1021" alt="Screen Shot 2022-06-24 at 11 04 55 AM" src="https://user-images.githubusercontent.com/103233915/175574224-4fbe523d-cce7-4691-88c5-2798892fbbcb.png">


检查了一下，vercel与github account关联成功了，确实帮我建立了一个新的private repo，但是这个repo是空白的，没有把waline的repo克隆过来。不知道这是什么原因？

**Reply by lizheming** [reply]
不太清楚，建议直接 vercel 工单咨询
